app/assets/stylesheets/components/btn.css.scss

```
@@ -291,3 +291,35 @@ form.button_to {
 .option-btn:hover {
   @include shadow-z4;
 }
+
+.single-zero-button, #zero-button{
```

```suggestion
.single-zero-button, #zero-button {
```
(Same for the other braces.)